polylogue/util.py

```
@@ -1,13 +1,13 @@
 import json
 import os
 import re
-import sys
-import datetime
-import difflib
 import subprocess
+import sys
 import tempfile
 from pathlib import Path
-from typing import Optional, Any, Dict
+from typing import Any, Dict, Optional
```

Ruff (0.15.9)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)